hail/python/test/hail/helpers.py

```diff
@@ -130,6 +130,17 @@ def wrapper(func, *args, **kwargs):
 
     return wrapper
 
+def skip_when_spark_backend():
```

Comment:
should this be a `fails_spark_backend` with xfail? because I expect this to pass once we turn on eager lowering.